<!DOCTYPE html>
<html lang="en">

<link href="css/style.css" rel="stylesheet" />
<link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.slim.min.js"/></script>
<script type="text/javascript" src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"/></script>

<!-- Navigation -->
<nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
  <div class="container">
    <a class="navbar-brand" href="#">cSynBioSys Group</a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
    <div class="collapse navbar-collapse" id="navbarResponsive">
      <ul class="navbar-nav ml-auto">
        <li class="nav-item active">
          <a class="nav-link" href="index.html">Home</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="research.html">Research</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="papers.html">Papers
            <span class="sr-only">(current)</span>
          </a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="people.html">People</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="news.html">News</a>
        </li>
      </ul>
    </div>
  </div>
</nav>

<header class="masthead">
  <div class="container h-100">
    <div class="row h-100 align-items-center">
      <div class="col-12 text-center">
        <h1 class="font-weight-light">Papers</h1>
        <p class="lead">Find links to all our publications here.</p>
      </div>
    </div>
  </div>
</header>

<hr class="featurette-divider">

<div class="row">
  <div class="col-md-4">
    <div class="card mb-4 box-shadow">
      <div class="card-body">
        <h2>P7 Sex differences in macrophage-mediated endothelin-1 clearance</h2>
        <h3>G Sutton, A Czopek, T Krueger, F Menolascina, N Dhaun</h3>
        <p class="card-text">Introduction Hypertension is a major risk factor for cardiovascular and chronic kidney disease. However, its causes remains unclear in most cases. Our recent data suggests that macrophages contribute to, and protect from, hypertension by regulating the endothelin (ET) system [Czopek, Eur Heart J 2019]. ET-1 is the most potent endogenous vasoconstrictor with pro-inflammatory properties. We have shown that macrophage remove ET-1 through the ETB receptor (ETBR). As cardiovascular risk varies between men and women, here we explored if the expression of the ET system and ET-1 clearance varies between male and female mice and with macrophage phenotype. Method Bone marrow derived macrophage from 9–12 weeks old male (n=5) and female (n=7) C57B6J mice were cultured in vitro. LPS/IFNγ were used to polarise macrophage to a pro-inflammatory/M1 phenotype and IL-4/IL-13 for anti-inflammatory/M2 polarisation. Gene expression was determined using qPCR. ET-1 concentration was measured by ELISA. Macrophage were then exposed to 2 ng/mL fluorescently-labelled ET-1 (fET-1) and the concentration was measured by fluorescence spectrophotometry after 12 hours. Statistical analyses were performed using a two-way ANOVA with Sidak’s multiple comparison method. Results Unstimulated macrophagesMale macrophage expressed 2-fold greater ETBR than females (p<0.01). There was no difference in ppET-1 expression between the sexes. Neither male nor female macrophage showed net production of ET-1. Whereas, male macrophage cleared all extracellular ET-1, females cleared none (p<0.05). Polarised macrophages Both male and female macrophage expressed ETBR and ppET-1. Whereas ETBR levels were similar, male macrophage displayed 3-fold greater ppET-1 than females (p<0.05). Net ET-1 production was similar between male and female macrophage (∼20 pg/mL at 24 h). However, male macrophage cleared extracellular ET-1 whereas females did not (p<0.05). Both male and female macrophage expressed ETBR and ppET-1 similarly. Neither males nor females demonstrated net ET-1 production or ET-1 clearance. Conclusions Our data suggest a sex difference in ET system expression and in ET-1 clearance by macrophage. The increased clearance seen with male macrophage is consistent with increased ETBR expression relative to female macrophage. These differences may be important in the cardiovascular risk difference observed between men and women.</p>
        <div class="d-flex justify-content-between align-items-center">
          <div class="btn-group">
            <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class="col-md-4">
    <div class="card mb-4 box-shadow">
      <div class="card-body">
        <h2>Face Coverings and Respiratory Tract Droplet Dispersion</h2>
        <h3>L Bandiera, G Pavar, G Pisetta, S Otomo, E Mangano, J R Seckl, P Digard, E Molinari, F Menolascina, I M Viola</h3>
        <p class="card-text">Respiratory droplets are the primary transmission route for SARS-CoV-2, a principle which drives social distancing guidelines. Evidence suggests that virus transmission can be reduced by face coverings, but robust evidence for how mask usage might affect safe distancing parameters is lacking. Accordingly, we set out to quantify the effects of face coverings on respiratory tract droplet deposition. We tested an anatomically realistic manikin head which ejected fluorescent droplets of water and human volunteers, in speaking and coughing conditions without a face covering, or with a surgical mask or a single-layer cotton face covering. We quantified the number of droplets in flight using laser sheet illumination and UV-light for those that had landed at table height at up to 2 m. For human volunteers, expiratory droplets were caught on a microscope slide 5 cm from the mouth. Whether manikin or human, wearing a face covering decreased the number of projected droplets by less than 1000-fold. We estimated that a person standing 2 m from someone coughing without a mask is exposed to over 10 000 times more respiratory droplets than from someone standing 0.5 m away wearing a basic single-layer mask. Our results indicate that face coverings show consistent efficacy at blocking respiratory droplets and thus provide an opportunity to moderate social distancing policies. However, the methodologies we employed mostly detect larger (non-aerosol) sized droplets. If the aerosol transmission is later determined to be a significant driver of infection, then our findings may overestimate the effectiveness of face coverings.</p>
        <div class="d-flex justify-content-between align-items-center">
          <div class="btn-group">
            <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class="col-md-4">
    <div class="card mb-4 box-shadow">
      <div class="card-body">
        <h2>Information content analysis reveals desirable aspects of in vivo experiments of a synthetic circuit</h2>
        <h3>DG Cabeza, L Bandiera, E Balsa-Canto, F Menolascina</h3>
        <p class="card-text">Synthetic biology is predicated upon the paradigm of rationally engineering biological systems to enrich them with new functions. Departing from the path outlined by other engineering disciplines, synthetic biology has made limited use of mathematical models so far. Indeed, their laborious inference leverages on noisy data that often provide partial insights into the system behaviour. If the quality of data generated via a perturbation is a road-block to inference, how should we analyse the informativeness of a stimulus? Which are the factors that contribute to it? Here we combine ideas from System Identification and Bayesian inference to quantify the information content of in vivo experiments for the calibration of a deterministic model of the genetic toggle switch. Beyond establishing a link between Bayes- and Fisher Information-based inference approaches, we find a ~ 40% gain can be ascribed to the dynamical properties of the perturbation scheme. Our results hint at the importance of including Bayesian experimental design in the characterisation of synthetic circuits.</p>
        <div class="d-flex justify-content-between align-items-center">
          <div class="btn-group">
            <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
          </div>
        </div>
      </div>
    </div>
  </div>

  <hr class="featurette-divider">

  <div class="row">
    <div class="col-md-4">
      <div class="card mb-4 box-shadow">
        <div class="card-body">
          <h2>An automated Raman-based platform for the sorting of live cells by functional properties</h2>
          <h3>K S Lee, M Palatinszky, F C Pereira, J Nguyen, V I Fernandez, A J Mueller, F Menolascina, H Daims, D Berry, M Wagner, R Stocker</h3>
          <p class="card-text">Stable-isotope probing is widely used to study the function of microbial taxa in their natural environment, but sorting of isotopically labelled microbial cells from complex samples for subsequent genomic analysis or cultivation is still in its early infancy. Here, we introduce an optofluidic platform for automated sorting of stable-isotope-probing-labelled microbial cells, combining microfluidics, optical tweezing and Raman microspectroscopy, which yields live cells suitable for subsequent single-cell genomics, mini-metagenomics or cultivation. We describe the design and optimization of this Raman-activated cell-sorting approach, illustrate its operation with four model bacteria (two intestinal, one soil and one marine) and demonstrate its high sorting accuracy (98.3 ± 1.7%), throughput (200–500 cells h−1; 3.3–8.3 cells min−1) and compatibility with cultivation. Application of this sorting approach for the metagenomic characterization of bacteria involved in mucin degradation in the mouse colon revealed a diverse consortium of bacteria, including several members of the underexplored family Muribaculaceae, highlighting both the complexity of this niche and the potential of Raman-activated cell sorting for identifying key players in targeted processes.</p>
          <div class="d-flex justify-content-between align-items-center">
            <div class="btn-group">
              <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="col-md-4">
      <div class="card mb-4 box-shadow">
        <div class="card-body">
          <h2>Computational Methods Enabling Next-Generation Bioprocesses</h2>
          <h3>JR Banga, F Menolascina</h3>
          <p class="card-text">Synthetic biology—the engineering of cells to rewire the biomolecular networks inside them—has
witnessed phenomenal progress. In a little over a decade it has evolved from demonstrating
proof-of-concept gene circuits in bacteria to developing a whole new class of cell factories able to
produce compounds that would otherwise be prohibitively costly and/or unsustainable to obtain,
e.g., squalene.
However, despite a thriving community and some notable successes, the basic task of assembling
a predictable gene network from biomolecular parts is still a key challenge; it often takes many
months to produce a gene circuit with the desired behavior. Mathematical models and Model-Based
Systems Engineering methods offer a unique opportunity to address this issue and change fundamental
procedures in Synthetic Biology.
</p>
          <div class="d-flex justify-content-between align-items-center">
            <div class="btn-group">
              <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="col-md-4">
      <div class="card mb-4 box-shadow">
        <div class="card-body">
          <h2>A novel role for myeloid endothelin-B receptors in hypertension</h2>
          <h3>A Czopek, R Moorhouse, L Guyonnet, T Farrah, O Lenoir, E Owen, J Van Bragt, H M Costello, F Menolascina, V Baudrie, D J Webb, D C Kluth, M A Bailey, PL Tharaux, N Dhaun</h3>
          <p class="card-text">Aims
Hypertension is common. Recent data suggest that macrophages (Mφ) contribute to, and protect from, hypertension. Endothelin-1 (ET-1) is the most potent endogenous vasoconstrictor with additional pro-inflammatory properties. We investigated the role of the ET system in experimental and clinical hypertension by modifying Mφ number and phenotype.

Methods and results
In vitro, Mφ ET receptor function was explored using pharmacological, gene silencing, and knockout approaches. Using the CD11b-DTR mouse and novel mice with myeloid cell-specific endothelin-B (ETB) receptor deficiency (LysMETB−/−), we explored the effects of modifying Mφ number and phenotype on the hypertensive effects of ET-1, angiotensin II (ANG II), a model that is ET-1 dependent, and salt. In patients with small vessel vasculitis, the impacts of Mφ depleting and non-depleting therapies on blood pressure (BP) and endothelial function were examined. Mouse and human Mφ expressed both endothelin-A and ETB receptors and displayed chemokinesis to ET-1. However, stimulation of Mφ with exogenous ET-1 did not polarize Mφ phenotype. Interestingly, both mouse and human Mφ cleared ET-1 through ETB receptor mediated, and dynamin-dependent, endocytosis. Mφ depletion resulted in an augmented chronic hypertensive response to both ET-1 and salt. LysMETB−/− mice displayed an exaggerated hypertensive response to both ET-1 and ANG II. Finally, in patients who received Mφ depleting immunotherapy BP was higher and endothelial function worse than in those receiving non-depleting therapies.

Conclusion
Mφ and ET-1 may play an important role in BP control and potentially have a critical role as a therapeutic target in hypertension.</p>
          <div class="d-flex justify-content-between align-items-center">
            <div class="btn-group">
              <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
            </div>
          </div>
        </div>
      </div>
    </div>

    <hr class="featurette-divider">

    <div class="row">
      <div class="col-md-4">
        <div class="card mb-4 box-shadow">
          <div class="card-body">
            <h2>Information content analysis reveals desirable aspects of in vivo experiments of a synthetic circuit</h2>
            <h3>D Gómez Cabeza, L Bandeira, E Balsa-Canto, F Menolascina</h3>
            <p class="card-text">Synthetic biology is predicated upon the paradigm of rationally engineering biological systems to enrich them with new functions. Departing from the path outlined by other engineering disciplines, synthetic biology has made limited use of mathematical models so far. Indeed, their laborious inference leverages on noisy data that often provide partial insights into the system behaviour. If the quality of data generated via a perturbation is a road-block to inference, how should we analyse the informativeness of a stimulus? Which are the factors that contribute to it? Here we combine ideas from System Identification and Bayesian inference to quantify the information content of in vivo experiments for the calibration of a deterministic model of the genetic toggle switch. Beyond establishing a link between Bayes- and Fisher Information-based inference approaches, we find a ~ 40% gain can be ascribed to the dynamical properties of the perturbation scheme. Our results hint at the importance of including Bayesian experimental design in the characterisation of synthetic circuits.</p>
            <div class="d-flex justify-content-between align-items-center">
              <div class="btn-group">
                <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class="col-md-4">
        <div class="card mb-4 box-shadow">
          <div class="card-body">
            <h2>Bayesian model selection in synthetic biology: factor levels and observation functions</h2>
            <h3>L Bandiera, DG Cabeza, E Balsa-Canto, F Menolascina</h3>
            <p class="card-text">Data-driven inference of the most plausible mechanistic model within a set of candidates is a major hurdle in synthetic and systems biology. Probabilistic model selection is hampered by limitations in the quality and amount of biological data. Furthermore, the computational cost of discriminating between competing models often leads the user to skip model selection and subjectively choose a model.

To challenge this practice, here we took a genetic toggle switch built in E. coli, considered three alternative models of it and used a Bayesian approach to rank these models based on the evidence from in vivo data. As the ranking depends on the information content of the data, we use Bayesian optimisation to design maximally informative inputs, i.e. chemical stimuli for the cells. We then explore how the optimality of such stimuli depended on the degrees of freedom in the optimisation (i.e. the number of segments in the input), showing a decrease of the attainable discriminatory power with the dynamic properties of the perturbation. We finally investigated the effect that the observable(s) selected in the optimisation exerts on the outcome of the latter. Our results suggest that Bayesian optimisation-based experimental design can be adopted as a means to discriminate between competing models of a gene regulatory network.</p>
            <div class="d-flex justify-content-between align-items-center">
              <div class="btn-group">
                <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class="col-md-4">
        <div class="card mb-4 box-shadow">
          <div class="card-body">
            <h2>A systematic framework for biomolecular system identification</h2>
            <h3>Z Tuza, L Bandiera, D Gomez-Cabeza, GB Stan, F Menolascina</h3>
            <p class="card-text">Unwieldy challenges, such as emergent behaviour
or lack of compositionality, hamper the rational engineering
of synthetic biomolecular systems. The use of mathematical
models would address many of these challenges, yet current
practices in synthetic biology make obtaining them time and
resource intensive. More importantly, in many cases, the process
of obtaining data for mathematical models is guided by intuition
rather than rigorous modelling requirements.
To make better use of the available resources, this tutorial
proposes an end-to-end framework for biomolecular system
identification. Given a biomolecular system inside a biological
organism, the framework leverages on system identification
techniques to automate the initial proposition of candidate
models for the system of interest. Then, statistical methods guide
the optimal design of experiments that enable discrimination
and calibration of the most plausible model candidates for the
underlying system.
We foresee that, by establishing a closed-loop between datagathering and model identification, the outlined approach
will accelerate and standardise computational modelling of
biomolecular systems</p>
            <div class="d-flex justify-content-between align-items-center">
              <div class="btn-group">
                <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
              </div>
            </div>
          </div>
        </div>
      </div>

      <hr class="featurette-divider">

      <div class="row">
        <div class="col-md-4">
          <div class="card mb-4 box-shadow">
            <div class="card-body">
              <h2>Optimally designed vs intuition-driven inputs: the study case of promoter activity modelling</h2>
              <h3>L Bandiera, V Kothamachu, E Balsa-Canto, PS Swain, F Menolascina</h3>
              <p class="card-text">Synthetic biology is an emerging engineering discipline that aims at synthesising logical circuits into cells to accomplish new functions. Despite a thriving community and some notable successes, the basic task of assembling predictable gene circuits is still a key challenge. Mathematical models are uniquely suited to help solve this issue. Yet in biology they are perceived as expensive and laborious to obtain because low-information experiments have often been used to infer model parameters. How much additional information can be gained using optimally designed experiments? To tackle this question we consider a building block in Synthetic Biology, an inducible promoter in yeast S. cerevisiae. Using in vivo data we re-fit a mathematical model for such a system; we then compare in silico the quality of the parameter estimates when model calibration is done using typical (e.g. step inputs) and optimally designed experiments. We find that Optimal Experimental Design leads to ~70% improvement in the predictive ability of the inferred models. We conclude providing suggestions on how optimally designed experiments can be implemented in vivo.</p>
              <div class="d-flex justify-content-between align-items-center">
                <div class="btn-group">
                  <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="col-md-4">
          <div class="card mb-4 box-shadow">
            <div class="card-body">
              <h2>On-line optimal input design increases the efficiency and accuracy of the modelling of an inducible synthetic promoter</h2>
              <h3>L Bandiera, Z Hou, V B Kothamachu, E Balsa-Canto, P S Swain, F Menolascina</h3>
              <p class="card-text">Synthetic biology seeks to design biological parts and circuits that implement new functions in cells. Major accomplishments have been reported in this field, yet predicting a priori the in vivo behaviour of synthetic gene circuits is major a challenge. Mathematical models offer a means to address this bottleneck. However, in biology, modelling is perceived as an expensive, time-consuming task. Indeed, the quality of predictions depends on the accuracy of parameters, which are traditionally inferred from poorly informative data. How much can parameter accuracy be improved by using model-based optimal experimental design (MBOED)? To tackle this question, we considered an inducible promoter in the yeast S. cerevisiae. Using in vivo data, we re-fit a dynamic model for this component and then compared the performance of standard (e.g., step inputs) and optimally designed experiments for parameter inference. We found that MBOED improves the quality of model calibration by ∼60%. Results further improve up to 84% when considering on-line optimal experimental design (OED). Our in silico results suggest that MBOED provides a significant advantage in the identification of models of biological parts and should thus be integrated into their characterisation.</p>
              <div class="d-flex justify-content-between align-items-center">
                <div class="btn-group">
                  <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="col-md-4">
          <div class="card mb-4 box-shadow">
            <div class="card-body">
              <h2>The art of fusion: A synthetic approach to create cross-kingdom hybrids</h2>
              <h3>E Cachat, L Nelson, D Sachs, L Bandiera, A McDonald, E Szymanski, T Bates, F Menolascina, I Zurr, A Elfick, J Calvert, S Rosser, O Catts</h3>
              <p class="card-text">Lewis Thomas said of of cell fusion cell fusion: “it is the most unbiologic of all phenomena, violating the most fundamental myths of last century, for it denies the importance of specificity, integrity, and separateness in living things”(1).
Cross-kingdom fusion raises ontological, ethical and poetic poetic questions: how do multi -kingdom cell fusions challenge our categories and understandings of life? Where do they belong within both biological and cultural realms? How does their existence impact the environment and society?
In this interdisciplinary collaboration between artists (SymbioticA, University of Western Australia), scientists and social scientists (SynthSys, University of Edinburgh), we propose to create yeast-mammalian cell fusions, while exploring such questions through art.
We are engineering these fusion hybrids and observing their formation as they assemble into a new cell type. The work-in-progress was presented as an art exhibition at the Edinburgh International Science Festival in April 2018.</p>
              <div class="d-flex justify-content-between align-items-center">
                <div class="btn-group">
                  <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                </div>
              </div>
            </div>
          </div>
        </div>

        <hr class="featurette-divider">

        <div class="row">
          <div class="col-md-4">
            <div class="card mb-4 box-shadow">
              <div class="card-body">
                <h2>Modelling of the GAL1 Genetic Circuit in Yeast Using Three Equations</h2>
                <h3>CC Hsu, W Yu-Heng, F Menolascina, TEM Nordling</h3>
                <p class="card-text">Synthetic gene circuits can be used to modify and control existing biological processes and thus e.g. increase drug yields. Currently their use is hampered by the, largely, trial and error approach used to design them. Lack of reliable quantitative dynamical models of genetic circuits e.g. prevents the use of well established control design methods. We aim toward creation of a pipeline for automated closed-loop identification of dynamic models of synthetically engineered genetic circuits in microorganisms. As a step towards this aim, we here study modelling of the input-output behaviour of the yGIL337 strain of S. cerevisiae. In this strain expression of the fluorescent reporter can be turned on by growing the yeast in galactose and off by glucose. We perform parameter estimation on a system of three ordinary differential equations of Michaelis-Menten type based on in vivo data from a microfluidic experiment by Fiore et al. (2013) after redoing the data preprocessing. The parameter estimation is done using AMIGO2–a state of the art Matlab toolbox for iterative identification of dynamical models. We show that the goodness-of-fit of our model is comparable to the five models proposed by Fiore et al. and we hypothesise that the system is an adaptive feedback system.</p>
                <div class="d-flex justify-content-between align-items-center">
                  <div class="btn-group">
                    <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card mb-4 box-shadow">
              <div class="card-body">
                <h2>Logarithmic sensing in Bacillus subtilis aerotaxis</h2>
                <h3>F Menolascina, R Rusconi, V I Fernandez, S Smriga, Z Aminzare, E D Sontag, R Stocker</h3>
                <p class="card-text">Aerotaxis, the directed migration along oxygen gradients, allows many microorganisms to locate favorable oxygen concentrations. Despite oxygen’s fundamental role for life, even key aspects of aerotaxis remain poorly understood. In Bacillus subtilis, for example, there is conflicting evidence of whether migration occurs to the maximal oxygen concentration available or to an optimal intermediate one, and how aerotaxis can be maintained over a broad range of conditions. Using precisely controlled oxygen gradients in a microfluidic device, spanning the full spectrum of conditions from quasi-anoxic to oxic (60 n mol/l–1 m mol/l), we resolved B. subtilis’ ‘oxygen preference conundrum’ by demonstrating consistent migration towards maximum oxygen concentrations (‘monotonic aerotaxis’). Surprisingly, the strength of aerotaxis was largely unchanged over three decades in oxygen concentration (131 n mol/l–196 μ mol/l). We discovered that in this range B. subtilis responds to the logarithm of the oxygen concentration gradient, a rescaling strategy called ‘log-sensing’ that affords organisms high sensitivity over a wide range of conditions. In these experiments, high-throughput single-cell imaging yielded the best signal-to-noise ratio of any microbial taxis study to date, enabling the robust identification of the first mathematical model for aerotaxis among a broad class of alternative models. The model passed the stringent test of predicting the transient aerotactic response despite being developed on steady-state data, and quantitatively captures both monotonic aerotaxis and log-sensing. Taken together, these results shed new light on the oxygen-seeking capabilities of B. subtilis and provide a blueprint for the quantitative investigation of the many other forms of microbial taxis.</p>
                <div class="d-flex justify-content-between align-items-center">
                  <div class="btn-group">
                    <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card mb-4 box-shadow">
              <div class="card-body">
                <h2>In-vivo identification and control of aerotaxis in Bacillus subtilis</h2>
                <h3>F Menolascina, R Stocker, ED Sontag</h3>
                <p class="card-text">Problems of identification and control of biological systems have recently attracted an increasing amount of attention. Most work in this field has classically focused either on gene or protein networks. In this manuscript, we focus on the control of a behavioral trait emerging from a signaling network: aerotaxis, the directed motion of bacteria towards (or away from) oxygen. To do so, we consider a bacterium, Bacillus subtilis, which is strongly attracted by oxygen, and we quantitatively probe the dynamics of accumulation of populations of this microorganism when exposed to tightly controlled gradients of oxygen generated in a microfluidic device. Combining in-vivo experiments with system identification methods, we determine a simple model of aerotaxis in B. subtilis, and we subsequently employ this model in order to compute the sequence of oxygen gradients needed to achieve regulation of the center of mass of the bacterial population. We then successfully validate both the model and the control scheme, by showing that in-vivo positioning control can be achieved via the application of the precomputed inputs in an open-loop configuration.</p>
                <div class="d-flex justify-content-between align-items-center">
                  <div class="btn-group">
                    <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <hr class="featurette-divider">

          <div class="row">
            <div class="col-md-4">
              <div class="card mb-4 box-shadow">
                <div class="card-body">
                  <h2>Online model selection for synthetic gene networks</h2>
                  <h3>W Pan, F Menolascina, GB Stan</h3>
                  <p class="card-text">Control algorithms combined with microfluidic devices and microscopy have enabled in vivo real-time control of protein expression in synthetic gene networks. Most control algorithms rely on the a priori availability of mathematical models of the gene networks to be controlled. These models are typically black/grey box models, which can be obtained through the use of data-driven techniques developed in the context of systems identification. Data-driven inference of both model structure and parameters is the main focus of this paper. There are two main challenges associated with the inference of dynamical models for real-time control of gene regulatory networks in living cells. Since biological systems are typically evolving over time, the first challenge stems from the fact that model selection needs to be done online, which prevents the application of computationally expensive identification algorithms iterating through large amounts of streaming data. The second challenge consists in performing nonlinear model selection, which is typically too burdensome for Kalman filtering related techniques due the heterogeneity and nonlinearity of the candidate models. In this paper, we combine sparse Bayesian techniques with classic Kalman filtering techniques to tackle these challenges.</p>
                  <div class="d-flex justify-content-between align-items-center">
                    <div class="btn-group">
                      <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="card mb-4 box-shadow">
                <div class="card-body">
                  <h2>Speed-dependent chemotactic precision in marine bacteria</h2>
                  <h3>K Son, F Menolascina, R Stocker</h3>
                  <p class="card-text">Chemotaxis underpins important ecological processes in marine bacteria, from the association with primary producers to the colonization of particles and hosts. Marine bacteria often swim with a single flagellum at high speeds, alternating "runs" with either 180° reversals or ∼90° "flicks," the latter resulting from a buckling instability of the flagellum. These adaptations diverge from Escherichia coli's classic run-and-tumble motility, yet how they relate to the strong and rapid chemotaxis characteristic of marine bacteria has remained unknown. We investigated the relationship between swimming speed, run-reverse-flick motility, and high-performance chemotaxis by tracking thousands of Vibrio alginolyticus cells in microfluidic gradients. At odds with current chemotaxis models, we found that chemotactic precision-the strength of accumulation of cells at the peak of a gradient-is swimming-speed dependent in V. alginolyticus Faster cells accumulate twofold more tightly by chemotaxis compared with slower cells, attaining an advantage in the exploitation of a resource additional to that of faster gradient climbing. Trajectory analysis and an agent-based mathematical model revealed that this unexpected advantage originates from a speed dependence of reorientation frequency and flicking, which were higher for faster cells, and was compounded by chemokinesis, an increase in speed with resource concentration. The absence of any one of these adaptations led to a 65-70% reduction in the population-level resource exposure. These findings indicate that, contrary to what occurs in E. coli, swimming speed can be a fundamental determinant of the gradient-seeking capabilities of marine bacteria, and suggest a new model of bacterial chemotaxis.</p>
                  <div class="d-flex justify-content-between align-items-center">
                    <div class="btn-group">
                      <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="card mb-4 box-shadow">
                <div class="card-body">
                  <h2>Aerotactic Cell Density Variations in Bacterial Turbulence</h2>
                  <h3>V Fernandez, S Smriga, F Menolascina, R Rusconi, R Stocker</h3>
                  <p class="card-text">Concentrated suspensions of motile bacteria such as Bacillus subtilis exhibit group dynamics much larger than the scale of an individual bacterium, visual similar to high Reynolds number turbulence. These suspensions represent a microscale realization of active matter. Individually, B. subtilis are also aerotactic, and will accumulate near oxygen sources. Using a microfluidic device for generating oxygen gradients, we investigate the relationship between individuals' attraction to oxygen and the collective motion resultant from hydrodynamic interactions. We focus on changes in density revealed by a fluorescently labeled sub-population of B. subtilis in the dense suspension. This approach allows us to examine changes in density during the onset of collective motion as well as fully developed bacterial turbulence.</p>
                  <div class="d-flex justify-content-between align-items-center">
                    <div class="btn-group">
                      <button type="button" class="btn btn-sm btn-outline-secondary">Link</button>
                    </div>
                  </div>
                </div>
              </div>
            </div>

</html>
